hack/crds-sync.sh

```diff
@@ -11,4 +11,4 @@ for crd in "${CRDS_MAPPING[@]}" ; do
     SRC="${crd%%:*}"
     DES="${crd##*:}"
     cp "$SRC" "install/$DES"
-done
+done
```

Comment:
Please revert

Reply:
Good catch, i thought i removed that.  My IDE annoys me sometimes.